providers/song-info.js

```diff
@@ -61,7 +61,8 @@ const handleData = async (responseText, win) => {
 		songInfo.album = data?.videoDetails?.album; // Will be undefined if video exist
 
 		const oldUrl = songInfo.imageSrc;
-		songInfo.imageSrc = videoDetails.thumbnail?.thumbnails?.pop()?.url.split("?")[0];
```

Comment:
Wanted to use `at.(-1) but it requires node 16.6+ aka electron 23+

Reply:
Well it has been released since then
https://www.electronjs.org/blog/electron-23-0

but it drops support for Windows 7/8/8.1, so I'm unsure when should we bump that